docs/contributing.md

```diff
@@ -41,9 +41,10 @@ git clone git@github.com:<your username>/pydantic.git
 cd pydantic
 
 # 2. Set up a virtualenv for running tests
-virtualenv -p `which python3.7` env
+virtualenv -p `which python3.8` env
 source env/bin/activate
-# (or however you prefer to setup a python environment, 3.6 will work too)
+# Building docs require 3.8. Otherwise, feel free to setup a python environment
+# however you prefer; 3.6 will work too.
```

Comment:
```suggestion
# Building docs requires 3.8. Otherwise, feel free to setup a python environment
# however you prefer; 3.6 will work too.
```
And I'm not a native so I may be wrong but the second sentence feels super weird to me.
What about something like "_If you don't need to build docs you can use whichever version; 3.6 will work too._"  ?